book/how_tos/data_access/CryoCloudScratchBucket.ipynb

```diff
@@ -0,0 +1,1480 @@
+       " 'type': 'file',\n",
+       " 'StorageClass': 'STANDARD',\n",
+       " 'VersionId': None,\n",
+       " 'ContentType': 'application/x-hdf5'}"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "s3.stat(remote_object)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29e6e602-1105-44a6-83b2-2903282c0578",
+   "metadata": {
+    "user_expressions": []
+   },
+   "source": [
+    "### Directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e9cf6509-b3ee-4962-8fa9-77277a70389f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 6.2G\n",
+      "-rw-r--r-- 1 jovyan jovyan 1015M Jul 20 15:29 ATL03_20230103090928_02111806_006_01.h5\n",
+      "-rw-r--r-- 1 jovyan jovyan  5.2G Jul 20 15:32 ATL03_20230108204519_02951802_006_01.h5\n"
+     ]
+    }
+   ],
+   "source": [
+    "local_dir = '/tmp/example_ATL03'\n",
+    "\n",
+    "!ls -lh {local_dir}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "8f50f4ef-78e1-48e2-9d13-78524b06c237",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[None, None]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "remote_prefix = f\"{os.environ['SCRATCH_BUCKET']}/example_ATL03\"\n",
+    "\n",
+    "s3.upload(local_dir, remote_prefix, recursive=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "f2e159f0-625a-44c3-8d80-55803a4287a7",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "s3://nasa-cryo-scratch/scottyhq/example_ATL03\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['nasa-cryo-scratch/scottyhq/example_ATL03/ATL03_20230103090928_02111806_006_01.h5',\n",
+       " 'nasa-cryo-scratch/scottyhq/example_ATL03/ATL03_20230108204519_02951802_006_01.h5',\n",
+       " 'nasa-cryo-scratch/scottyhq/example_ATL03/example_ATL03']"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "print(remote_prefix)\n",
+    "s3.ls(remote_prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eced7ea8-79e7-4bcf-a723-3221eec6018e",
+   "metadata": {
+    "user_expressions": []
+   },
+   "source": [
+    "## Accessing Data\n",
+    "\n",
+    "Some software packages allow you to stream data directly from S3 Buckets. But you can always pull objects from S3 and work with local file paths. For file formats that are not Cloud Optimized (like HDF!) this often gives the best performance. \n",
```

Comment:
For file formats (like HDF!) that are not cloud-optimized this often gives the best performance.

I confused this as saying HDF was cloud optimized at first